### PR TITLE
Zero root domain static site support

### LINKED ISSUE #229: Zero create step fails on templating a non-text file located in a directory
Running `zero create` crashes while templating non-text files; this happens if your zero template contains an image  / non-text file in a dir that isn't ignored.
 https://github.com/commitdev/zero/blob/b253a1ef75541b29c6aaec5edc93f31da9890256/internal/generate/generate_modules.go#L77-L88

**Possible Solutions:**
have another condition for 
- known binary file extensions
- read files header -> copy the file over if it's !type text.

related to #222

### PR DESCRIPTION
**Pull request by dtoki**
**Adds zero landing page registry**

- add a new repo to zero registry

**Don't template binary files.**

- Considering the definition of a binary file as "a computer file that is not a text file", they can be differentiated by searching for the text/plain MIME in its MIME hierarchy.

closes #229

### CODE REVIEW

File: internal/generate/generate_modules.go [hunk 1]
```diff
@@ -65,9 +74,17 @@ type TemplateConfig struct {
 	isTemplate  bool
 }
 
+type BinFileConfig struct {
+	source      string
+	destination string
+	mineType    string
```
Comment (bmonkman) [inline internal/generate/generate_modules.go:80]: Typo

Comment (bmonkman) [reply internal/generate/generate_modules.go:80]: Any reason to have the mime type here? Or just because it was available?

Comment (dtoki) [reply internal/generate/generate_modules.go:80]: just because it's available; if we'd like to post-process on filter on it in the future.

Comment (bmonkman) [reply internal/generate/generate_modules.go:80]: Actually "isTemplate" in TemplateConfig is not even used anymore. How about we just have a struct with source and destination and use that for both return lists?

File: internal/generate/generate_modules.go [hunk 2]
```diff
@@ -80,6 +97,29 @@ func newTemplates(moduleDir string, outputDir string, overwrite bool) []*Templat
...
+		if isBinary {
+			binFile = append(binFile, &BinFileConfig{
+				source:      path,
+				destination: fs.ReplacePath(path, moduleDir, outputDir),
+				mineType:    detectedMIME.String(),
+			})
+			continue
+		}
+
 		_, file := filepath.Split(path)
```
Comment (bmonkman) [inline internal/generate/generate_modules.go:123]: Mind removing this block? We don't use it anymore, as we just try to template everything in the repo (that's not binary)

**Pull request merged**